packages/styles/gulp/tasks/sass.js

```diff
@@ -43,11 +41,7 @@ function _sass() {
     )
     .pipe(rename(global.config.distCss))
     .pipe(gulp.dest('dist'))
-    .pipe(
-      cleanCSS({
-        level: 2,
-      })
-    )
+    .pipe(cleanCSS())
```

Comment:
Do we have raw CSS code? Otherwise `outputStyle: compressed` in Sass compilation may further improve the build time.